### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by Kumilachew-g**
**In this PR, We have done unit tests for the following:**

- Calculator component Test
- Home component Test
- Quotes component Test
- Calculator logic functions (Operate and Calculate) Test
- Calculator Particles Test
- NotMatch Component Test
- Navbar Component Test
- Particles Component Test
In addition to these, We have checked the calculator is functioning correctly after testing.
**Testing Screenshots:**
image
**Calculator Home Page:**
image
**Calculator Page:**
image
**Quote Page**
image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -26,6 +27,7 @@ const App = () => (
       </Route>
     </Switch>
   </Router>
+  </React.StrictMode>
 );
 
 export default App;
```
Comment (zieeco) [inline src/App.js]: > ***NOTE: This change is only applicable to the import statement. I'm using this inline comment to be able to add a review. please refer to the import statement.***

- Hey Team, great job with the test, however, the test is failing due to how you've wrongly imported the calculator components. 
- As you used a lowercase letter for the `c` which you've imported as an Uppercase `C`. For this reason, your test are not passing. Please kindly fix this issue to make the test pass.

`import Calculator from './components/calculator';
`

File: src/_tests__/Calculator.test.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import '@testing-library/jest-dom/extend-expect';
+import { render, fireEvent } from '@testing-library/react';
+/* eslint-disable */
+import Calculator from '../components/Calculator';
```
Comment (zieeco) [inline src/_tests__/Calculator.test.js]: - The same issue is applicable here.

Comment (zieeco) [reply src/_tests__/Calculator.test.js]: - The same issue is applicable to your Calculator components where you've imported the 
`import './Calculator.css';` which should be `import './calculator.css';
`

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Kumilachew-g  & @Agetuni,


Great job so far! 👏 
You've done it well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @Kumilachew-g , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive PR.
:heavy_check_mark: Well implemented requirements.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "fix bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**